Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ImageEx/ImageExLazyLoadingControl.xaml

```diff
@@ -0,0 +1,42 @@
+﻿<UserControl x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.ImageExLazyLoadingControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             d:DesignHeight="300"
+             d:DesignWidth="400"
+             mc:Ignorable="d">
+    <Grid Background="#80000000">
+        <Grid Margin="40"
+              Background="White">
+            <ScrollViewer VerticalScrollBarVisibility="Auto">
+                <Grid Height="3000">
+                    <Border Width="200"
+                            Height="200"
+                            HorizontalAlignment="Center"
+                            VerticalAlignment="Center"
+                            BorderBrush="Black"
+                            BorderThickness="1">
+                        <controls:ImageEx EnableLazyLoading="True"
+                                          ImageExOpened="ImageEx_ImageExOpened"
+                                          IsCacheEnabled="False"
+                                          Source="/Assets/Photos/LunchBreak.jpg" />
+                    </Border>
+                </Grid>
+            </ScrollViewer>
+            <TextBlock HorizontalAlignment="Left"
+                       VerticalAlignment="Top"
+                       Foreground="OrangeRed"
+                       IsHitTestVisible="False"
+                       Text="Please scroll down to see the effect." />
+            <AppBarButton Width="48"
```

Comment:
@h82258652 I'm not seeing the close button?
![image](https://user-images.githubusercontent.com/24302614/52088693-6e0e5800-2561-11e9-8b79-28316f0cb358.png)

Should we explicitly create some rows to ensure the appbar is visible?

Reply:
That's strange, I can see the close button on my PC. I will use a ```Button``` instead the ```AppBarButton```.

Reply:
Done.